Strategy11/formidable-forms
# Fix Captcha not rendering in bs modal

Fix https://github.com/Strategy11/formidable-modal/issues/46

### Relevant knowledge base
https://formidableforms.com/knowledgebase/bootstrap-modal/

### Test Procedure
1. Go to your Formidable Global settings > Catpcha, then configure and allow multiple captchas like this:
![CleanShot 2024-10-17 at 09 30 03@2x](https://github.com/user-attachments/assets/185aebcb-675a-436a-8706-951393d069cd)
You can create reCaptcha keys here: https://www.google.com/recaptcha/
2. Create a form and add fields to it.
3. Create a WordPress page and add `Formidable Modal` block, select the block and add a `Formidable Form` block to the modal block. Then select the form you just created from the `Formidable Form` block options on the side bar.
4. Save and Preview the wordpress page.
5. Trigger the modal by clicking the modal trigger button.
6. Confirm that the captcha is loaded in your form as in this screenshot and you should be able to submit the form.
<img width="956" alt="image" src="https://github.com/user-attachments/assets/7237ef17-e725-446e-b8a8-9acd59c1188d">
